### PR TITLE
add get message

### PR DESCRIPTION
**Pull request by rileylnapier**
## Description of the change

add getMessage for /messages/{messageId} api

## Type of change
- [ ] Bug fix (non-breaking change that fixes an issue)
- [x] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Related issues

> Fix [#1]() 

### Development

- [x] Lint rules pass locally
- [x] The code changed/added as part of this pull request has been covered with tests
- [x] All tests related to the changed code pass in development

### Code review 

- [x]  This pull request has a descriptive title and information useful to a reviewer. There may be a screenshot or screencast attached
- [x] "Ready for review" label attached to the PR and reviewers mentioned in a comment
- [ ] Changes have been reviewed by at least one other engineer
- [ ] Issue from task tracker has a link to this pull request

### CODE REVIEW

**Comment by aydrian** [comment]
I think it looks good, but we've introduced trailing commas in to the ReadMe examples. Is regular JavaScript okay with trailing commas? I think I would prefer to not have them there. Thoughts?

**Comment by rileylnapier** [comment]
> I think it looks good, but we've introduced trailing commas in to the ReadMe examples. Is regular JavaScript okay with trailing commas? I think I would prefer to not have them there. Thoughts?

my editor added them :(.

javascript does seem to support them

```JavaScript has allowed trailing commas in array literals since the beginning, and later added them to object literals (ECMAScript 5) and most recently (ECMAScript 2017) to function parameters.```

**Comment by aydrian** [comment]
I figured. We should probably look to add an editor.config or something.
It's probably fine then. I just want to make sure if people copy the example, it works/doesn't introduce errors with the understanding that some placeholder data will need to be replaced.

File: src/types.ts [hunk 1]
```diff
@@ -66,6 +66,28 @@ export interface ICourierProfileGetResponse {
   profile: object;
 }
 
+export interface ICourierMessageGetResponse {
+  enqueued?: number,
```
Comment (scarney81) [inline src/types.ts:70]: Consistent with semi-colon and ,

**Pull request merged**